forms-flow-web/src/components/FOI/customComponents/ContactApplicant/util.tsx

```diff
@@ -31,7 +30,12 @@ export const getExtensiondetails = (requestExtensions:any, type: string) => {
     return ["","","","","","",""]
 }
 
-export const getExtensionType = (requestExtensions: any) => {
+export const getExtensionType = (requestDetails:any, requestExtensions: any) => {
+
+  if (requestDetails.currentState === "Open") {
+    return "NA";
+  }
```

Comment:
Will we need to do this for the other 3 states (intake in progress, unopened, app fee owing)?

Reply:
Good point! we need to show the OIPC and PB templates in states beyond 'Open', so it seems like a necessary addition.